src/shims/time.h

```diff
@@ -108,12 +108,13 @@ _dispatch_get_nanoseconds(void)
 	return _dispatch_timespec_to_nano(ts);
 #elif defined(_WIN32)
 	// FILETIME is 100-nanosecond intervals since January 1, 1601 (UTC).
+	static const uint64_t kNTToUNIXBiasAdjustment = 11644473600 * NSEC_PER_SEC;
 	FILETIME ft;
 	ULARGE_INTEGER li;
-	GetSystemTimeAsFileTime(&ft);
+	GetSystemTimePreciseAsFileTime(&ft);
 	li.LowPart = ft.dwLowDateTime;
 	li.HighPart = ft.dwHighDateTime;
-	return li.QuadPart * 100ull;
+	return li.QuadPart * 100ull + kNTToUNIXBiasAdjustment;
```

Comment:
Shouldn't we be subtracting the bias here rather than adding?

Reply:
Yeah, I thought I was subtracting.